Sming/SmingCore/Network/TcpConnection.cpp

```diff
@@ -297,8 +298,8 @@ void TcpConnection::close()
 #ifdef ENABLE_SSL
 	if (ssl != nullptr) {
 		debugf("SSL: closing ...");
-//		ssl_ctx_free(sslContext);
 		ssl_free(ssl);
```

Comment:
@ericogr Same as above - delete that line above. The line below should take care to do the right free-ing.